worker/tests/unit/fastlyJob.test.js

```diff
@@ -24,8 +24,20 @@ const map = {
   'target': 'target'
 };
 
+// test purge urls
+const opsmanagerUrls = [
+  'https://docs.opsmanager.mongodb.com/current/installation/',
+  'https://docs.opsmanager.mongodb.com/current/core/requirements/',
+  'https://docs.mongodb.com/drivers/cxx/'
```

Comment:
I added a drivers url to see if this works for all properties. I think it would be good to add a bunch of other urls for valid pages that are unlikely to change across the different properties. We also need to add a test (or a few tests) to test failures -- urls that don't exist, for example. This is to make sure we recover from these scenarios properly.